lib/vehicle_factory.rb

```diff
@@ -0,0 +1,17 @@
+class VehicleFactory
+  def initialize
+   
+  end
+
+  def create_vehicles(api_data)
+    @factory_vehicles = []
+    api_data.each do |data|
+      vehicle_data = { vin: data[:vin_1_10], make: data[:make], 
+                      model: data[:model], year: data[:model_year], 
+                      engine: 'ev' }
+      vehicle = Vehicle.new(vehicle_data)
+      @factory_vehicles << vehicle
+    end
+    @factory_vehicles
+  end
```

Comment:
When declaring a hash, it is much easier to read when all the keys are on their own line.

```ruby
def create_vehicles(api_data)
  @factory_vehicles = []
  api_data.each do |data|
    vehicle_data = {
      vin: data[:vin_1_10], 
      make: data[:make], 
      model: data[:model], 
      year: data[:model_year], 
      engine: 'ev' # This is supposed to be :ev, not 'ev'
    }
    vehicle = Vehicle.new(vehicle_data)
    @factory_vehicles << vehicle
  end
  @factory_vehicles
end
```

Finally, instead of manually creating an array, `@factory_vehicles`, to store the vehicles and then returning it at the end of the method, you can simplify the code by using the `map` method to transform the `api_data` into an array of `Vehicle` objects and directly return it:

```ruby
def create_vehicles(api_data)
  api_data.map do |data|
    vehicle_data = {
      vin: data[:vin_1_10], 
      make: data[:make], 
      model: data[:model], 
      year: data[:model_year], 
      engine: 'ev' 
    }
    
    Vehicle.new(vehicle_data)
  end
end
```